src/styles/main.scss

```diff
@@ -0,0 +1,120 @@
+$watch-size: 400px;
+$pin-size: $watch-size/30;
+
+@mixin watch-hand($w, $h, $clr, $br-clr, $spin-dur) {
+  position: absolute;
+  bottom: 50%;
+
+  box-sizing: border-box;
+  width: $w;
+  height: $h;
+  border-radius: $w/2;
+  border: 1px solid $br-clr;
+
+  background-color: $clr;
+
+  transform-origin: bottom;
+  animation: spin $spin-dur steps(60, end);
+  animation-play-state: paused;
+  animation-iteration-count: infinite;
+}
+
+@mixin sircle ($d) {
+  width: $d;
+  height: $d;
+  border-radius: $d/2;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  height: 100vh;
+  margin: 0;
+
+  font-family: Courier, monospace;
```

Comment:
Always set fonts in the `html {...}` tag

Reply:
done